lib/msf/core/db_manager/connection.rb

```diff
@@ -18,7 +18,7 @@ def after_establish_connection
       migrate
 
       # Set the default workspace
-      self.workspace = self.default_workspace
+      #framework.db.workspace = framework.db.default_workspace
```

Comment:
Should this line be commented out? If so, remove the line and the comment on the line above.